### ISSUE TITLE
not working on firefox

### ISSUE THREAD

**Issue by Yoposoc** [dnsev/4cs#74]
http://puu.sh/mNS8F/b41c53a9df.jpg
Yeah... not working right now on firefox 44.0 and GM3.6. I thought it was appchanx so I turned it off but it still doesn't show.
Here's my console: 
http://puu.sh/mNRYH/df4e74282d.png
I'm thinking it's that error. The first and last 3 lines are from when I refreshed the thread.

**Comment by dnsev** [comment]
I don't think this has ever worked on Greasemonkey 2.x+, as it was abandoned around when it came out.

This software is not maintained anymore, and is likely to only "work" on older versions of browsers. (If you look at the source code, you'll understand why. [hint: it's a disaster])

I'm not really sure why the creator(s) of these threads advertise this site as having a working script; it's very misleading.

**Comment by Yoposoc** [comment]
Huh, I see. Thanks.
